src/sig_stfl/sig_stfl.c

```diff
@@ -0,0 +1,1435 @@
+		return OQS_SIG_STFL_alg_lms_sha256_h15_w8_h15_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h5_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h5_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h5_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h10_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h10_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h10_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h15_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h15_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h15_w8_new();
+#else
+		return NULL;
+#endif
+	} else if (0 == strcasecmp(method_name, OQS_SIG_STFL_alg_lms_sha256_h20_w8_h20_w8)) {
+#ifdef OQS_ENABLE_SIG_STFL_lms_sha256_h20_w8_h20_w8
+		return OQS_SIG_STFL_alg_lms_sha256_h20_w8_h20_w8_new();
+#else
+		return NULL;
+#endif
+	}
+//#endif //OQS_ENABLE_SIG_STFL_LMS
+	else {
+		return NULL;
+	}
+}
+
+OQS_API OQS_STATUS OQS_SIG_STFL_keypair(const OQS_SIG_STFL *sig, uint8_t *public_key, OQS_SIG_STFL_SECRET_KEY *secret_key) {
+#ifndef OQS_ALLOW_SFTL_KEY_AND_SIG_GEN
+	(void)sig;
+	(void)public_key;
+	(void)secret_key;
+	return OQS_ERROR;
+#else
+	if (sig == NULL || sig->keypair == NULL || sig->keypair(public_key, secret_key) != 0) {
+		return OQS_ERROR;
+	} else {
+		return OQS_SUCCESS;
+	}
+	return OQS_ERROR;
+#endif
+}
+
+OQS_API OQS_STATUS OQS_SIG_STFL_sign(const OQS_SIG_STFL *sig, uint8_t *signature, size_t *signature_len, const uint8_t *message,
+                                     size_t message_len, OQS_SIG_STFL_SECRET_KEY *secret_key) {
+#ifndef OQS_ALLOW_SFTL_KEY_AND_SIG_GEN
+	(void)sig;
+	(void)signature;
+	(void)signature_len;
+	(void)message;
+	(void)message_len;
+	(void)secret_key;
+	return OQS_ERROR;
+#else
+	if (sig == NULL || sig->sign == NULL || sig->sign(signature, signature_len, message, message_len, secret_key) != 0) {
+		return OQS_ERROR;
+	} else {
+		return OQS_SUCCESS;
+	}
+#endif
+}
+
+OQS_API OQS_STATUS OQS_SIG_STFL_verify(const OQS_SIG_STFL *sig, const uint8_t *message, size_t message_len, const uint8_t *signature, size_t signature_len, const uint8_t *public_key) {
+	if (sig == NULL || sig->verify == NULL || sig->verify(message, message_len, signature, signature_len, public_key) != 0) {
+		return OQS_ERROR;
+	} else {
+		return OQS_SUCCESS;
+	}
+}
+
+OQS_API OQS_STATUS OQS_SIG_STFL_sigs_remaining(const OQS_SIG_STFL *sig, unsigned long long *remain, const OQS_SIG_STFL_SECRET_KEY *secret_key) {
+#ifndef OQS_ALLOW_SFTL_KEY_AND_SIG_GEN
+	(void)sig;
+	(void)remain;
+	(void)secret_key;
+	return OQS_ERROR;
+#else
+	if (sig == NULL || sig->sigs_remaining == NULL || sig->sigs_remaining(remain, secret_key) != 0) {
+		return OQS_ERROR;
+	} else {
+		return OQS_SUCCESS;
+	}
+#endif //OQS_ALLOW_SFTL_KEY_AND_SIG_GEN 
```

Comment:
```suggestion
#endif //OQS_ALLOW_STFL_KEY_AND_SIG_GEN 
```

Reply:
done in https://github.com/open-quantum-safe/liboqs/pull/1650/commits/fbb09d1b4eafc14fe231dbe8da8c4d60b9bdcd79